src/controllers/auth.js

```diff
@@ -1,6 +1,6 @@
 import HttpStatus from 'http-status-codes';
 
-import * as authService from '../services/authService';
+import * as authService from '@/services/authService';
```

Comment:
@Abhigyandhakallf rather than this, can we change syntax to:

from 'services/authService', (auto import uses this syntax)
OR
from '@services/authService'?